src/js/easymde.js

```diff
@@ -2863,4 +2863,6 @@ EasyMDE.prototype.toTextArea = function () {
     }
 };
 
+EasyMDE.CodeMirror = CodeMirror;
```

Comment:
This isn't needed anymore, right?

Reply:
Not for your preferred approach, correct.  I'd still find it useful to avoid ~66K duplication though.